doc/Publishing.md

```diff
@@ -22,6 +22,13 @@ It's good to give a heads-up to the Theia developers some hours before a release
 Here is an [example](https://community.theia-ide.org/t/0-11-0-release/373).
 
 
+## Pre-Release Steps
+
+- Ensure that the [changelog](https://github.com/eclipse-theia/theia/blob/master/CHANGELOG.md) is updated and merged for the release.
+- Ensure that merged pull-requests for the given release are added to the corresponding release [milestone](https://github.com/eclipse-theia/theia/milestones):
```

Comment:
Is there a query you use to find the relevant PRs to check? If so, it is maybe worth to add a template URL here. Otherwise nvm...

Reply:
@JonasHelming I only look at the relevant [commits](https://github.com/eclipse-theia/theia/commits/master) which I know are not from committers. Perhaps there is a better way :smile: